packages/compass-aggregations/src/components/pipeline-builder-workspace/index.tsx

```diff
@@ -18,68 +15,26 @@ const containerStyles = css({
   height: '100%',
 });
 
-const workspaceStyles = css({
-  paddingBottom: spacing[3],
-  width: '100%',
-  overflow: 'auto',
-});
```

Comment:
This style that was critical to provide bottom padding, has been moved to individual workspaces